src/Sentry/SentryHttpMessageHandler.cs

```diff
@@ -61,18 +62,28 @@ public SentryHttpMessageHandler()
             // in case the user didn't set an inner handler.
             InnerHandler ??= new HttpClientHandler();
 
+            var requestMethod = request.Method.Method.ToUpperInvariant();
+            var url = request.RequestUri?.ToString() ?? string.Empty;
+
             // Start a span that tracks this request
             // (may be null if transaction is not set on the scope)
             var span = _hub.GetSpan()?.StartChild(
-                "http.client",
+            "http.client",
                 // e.g. "GET https://example.com"
-                $"{request.Method.Method.ToUpperInvariant()} {request.RequestUri}"
+                $"{requestMethod} {url}"
             );
 
             try
             {
                 var response = await base.SendAsync(request, cancellationToken).ConfigureAwait(false);
 
+                var breadcrumbData = new Dictionary<string, string>
+                {
+                    { "url", url },
+                    { "method", requestMethod }
+                };
+                _hub.AddBreadcrumb(string.Empty, "http", "http", breadcrumbData);
```

Comment:
Instead of passing `Empty` which will result in a `""` being sent to Sentry, why not go "manual" and do `new Breadcrumb`?
```suggestion
                _hub.AddBreadcrumb(string.Empty, "http", "http", breadcrumbData);
```